piquasso/_simulators/fock/general/state.py

```diff
@@ -318,3 +319,281 @@ def get_purity(self):
         np = self._connector.np
         density_matrix = self.density_matrix
         return np.real(np.einsum("ij,ji", density_matrix, density_matrix))
+
+    def plot_wigner_fockstate(
```

Comment:
A method of `FockState` - the `fockstate` suffix is redundant in the name.
```suggestion
    def plot_wigner(
```